packages/mui-lab/src/Timeline/Timeline.tsx

```diff
@@ -17,7 +17,7 @@ export interface TimelineProps extends StandardProps<React.HTMLAttributes<HTMLUL
    * The position where the TimelineContent should appear relative to the time axis.
    * @default 'right'
    */
-  position?: 'left' | 'right' | 'alternate';
+  position?: 'left' | 'right' | 'alternate' | 'alternate-revert';
```

Comment:
```suggestion
  position?: 'left' | 'right' | 'alternate' | 'alternate-reverse';
```

This would follow the convention we have in the `Grid` and `Stack` components for e.g. We would need to change all other usage of `revert` in the PR too.